java/java.hints/test/unit/src/org/netbeans/modules/java/hints/errors/VarCompDeclarationTest.java

```diff
@@ -68,16 +68,17 @@ public void testCase2() throws Exception {
                        "package test; \n" +
                        "public class Test {\n" +
                        "    private void test() { \n" +
-                       "        var v = 1, /*comment*/ v1 = 10;\n" +
+                       "        var v = 1, v1 = 10, v2 = 100;\n" +
```

Comment:
Will it not support compound var declaration with comment in between 2 variable?

Reply:
With recent nb-javac changes, the in between comment is not supported.

Reply:
hi @vikasprabhakar,  Is any work on review comments pending?

Reply:
Nothing is pending. Please approve and merge it.